generate-plugin.js

```diff
@@ -0,0 +1,134 @@
+'use strict'
+
+const {
+  readFile,
+  writeFile,
+  existsSync
+} = require('fs')
+const path = require('path')
+const chalk = require('chalk')
+const generify = require('generify')
+const argv = require('yargs-parser')
+const cliPkg = require('./package')
+const { execSync } = require('child_process')
+const log = require('./log')
+
+const pluginTemplate = {
+  dir: 'plugin',
+  main: 'index.js',
+  types: 'index.d.ts',
+  scripts: {
+    lint: 'standard && npm run lint:typescript',
+    'lint:typescript': 'standard --parser @typescript-eslint/parser --plugin @typescript-eslint/eslint-plugin test/types/*.ts',
+    test: 'npm run lint && npm run unit && npm run typescript',
+    typescript: 'tsd',
+    unit: 'tap test/**/*.test.js'
+  },
+  dependencies: {
+    'fastify-plugin': cliPkg.devDependencies['fastify-plugin']
+  },
+  devDependencies: {
+    '@types/node': cliPkg.devDependencies['@types/node'],
+    '@typescript-eslint/eslint-plugin': cliPkg.devDependencies['@typescript-eslint/eslint-plugin'],
+    '@typescript-eslint/parser': cliPkg.devDependencies['@typescript-eslint/parser'],
+    fastify: cliPkg.devDependencies.fastify,
+    standard: cliPkg.devDependencies.standard,
+    tap: cliPkg.devDependencies.tap,
+    tsd: cliPkg.devDependencies.tsd,
+    typescript: cliPkg.devDependencies.typescript
+  },
+  tsd: {
+    directory: 'test'
+  },
+  logInstructions: function (pkg) {
+    log('debug', 'saved package.json')
+    log('info', `project ${pkg.name} generated successfully`)
+    log('debug', `run '${chalk.bold('npm install')}' to install the dependencies`)
+    log('debug', `run '${chalk.bold('npm test')}' to execute the tests`)
```

Comment:
```suggestion
    log('debug', `run '${chalk.bold('npm run test')}' to execute the tests`)
```

Reply:
I would prefer to leave it as `npm test` since it is supported and widely use like this.